### ISSUE TITLE
RFC: Find a proper fix to lit-analyzer error with lit `ref()` directive

### ISSUE THREAD

**Issue by roberttran-cc** [CleverCloud/clever-components#679]
Usage of lit `ref()` triggers an error with lit-analyzer:
```
Unknown attribute '_________________'. Did you mean '_____________________________'?
323:  ${ref(__refRole)}
no-unknown-attribute
```

We've got [kind of a workaround](https://github.com/CleverCloud/clever-components/blob/master/tsconfig.json#L43-L48) but it's not really satisfying.

Context:
- same issue has been tracked [here](https://github.com/runem/lit-analyzer/issues/193) on the official repository
- Lit team has taken responsibility of the repository
- Lit team is working on its own [analyzer](https://github.com/lit/lit/tree/main/packages/labs/analyzer) which is still at early stage

**Comment by hsablonniere** [comment]
Decision:  @florian-sanders-cc will investigate the behavior of the problematic rule: "what do we miss if we disabled it?". With this info, we'll go for one of those:

* keep using the workaround
* disable the rule

**Comment by florian-sanders-cc** [comment]
It looks like the lit team has been working on lit-analyzer because the project is essential and the analyzer rework will take some time. (cf https://github.com/runem/lit-analyzer/issues/219)

The issues with `ref` and other Lit 2 directives have been fixed more than 1 year ago with the `2.0.0-pre.1` version (cf https://github.com/runem/lit-analyzer/issues/141 and https://github.com/runem/lit-analyzer/pull/172).

The `2.0.0-pre.3` version has been published a few days ago. I tested the orga-member components without the refs workaround and with this version of `lit-analyzer`, it no longer throws any errror.

Even if it's not a stable release, I'd say it's worth moving to it as I'm pretty sure it solves more issues than it could cause.
WDYT?
